src/apis/modTools.es6.js

```diff
@@ -0,0 +1,23 @@
+import apiRequest from '../apiBase/apiRequest';
+
+const parsePostBody = apiResponse => {
+  const { body } = apiResponse.response;
+
+  return apiResponse;
+};
+
+const del = (apiOptions, id, spam) => {
+    // Remove a link, comment, or modmail message.
+    const body = {id, spam};
+
+    return apiRequest(apiOptions, 'POST', 'api/remove', { body, type: 'form' }).then(parsePostBody);
+}
+
+const approve = (apiOptions, id) => {
+    // Approve a link or comment
+    const body = { id }
```

Comment:
We probably don't need this as its the only parameter, you could probably just pass this directly into `apiRequest` rather than create a `body` var and then pass it in.